zipkin-ui/README.md

```diff
@@ -70,3 +70,18 @@ and then you can submit the span(s) via curl:
 ```bash
 $ curl -H "Content-Type: application/json" --data @span.json http://localhost:9411/api/v1/spans 
 ```
+
+#### How do I find logs associated with a particular trace
+
+Since zipkin provides a correlation id (the trace id), it's a good pattern to add it in your logs.
+If you use zipkin, it's very likely that you have distributed logging sytem also and a way to query your logs (e.g. ELK stack).
+
+One convenient way to switch from a trace to its logs is to get a button which directly links a trace to its logs.
+
+This feature can be activated adding this environment variable:
+
+`LOGS_URL=http://kibana.company.com/query={traceId}`
```

Comment:
@marcingrzejszczak do you have an example kibana link that looks up from the id pattern sleuth logs with slf4j?